src-vue/src/views/DeveloperView.vue

```diff
@@ -119,6 +122,24 @@ export default defineComponent({
                 position: 'bottom-right'
             });
         },
+        async launchGameWithoutChecks() {
```

Comment:
Why did you duplicate the `launchGame` method?
I would keep the original method, and add a `skipChecks` flag as a parameter to it.

Reply:
You can simply add a parameter to the commit call
https://vuex.vuejs.org/guide/mutations.html#commit-with-payload

Reply:
Thanks for the link. Implemented with cced173115613ad68b49036937410578d5bb51ed